src/modules/navigator/rtl.cpp

```diff
@@ -692,11 +693,10 @@ float RTL::calculate_return_alt_from_cone_half_angle(float cone_half_angle_deg)
 		return_altitude_amsl = _destination.alt + 2.0f * destination_dist;
 
 	} else {
-
-		if (cone_half_angle_deg > 0.0f && destination_dist <= _param_rtl_min_dist.get()) {
+		if (cone_half_angle_deg != 0 && destination_dist <= _param_rtl_min_dist.get()) {
```

Comment:
Instead of passing it in as a integer and converting it when used below you could also just change the check here to > FLT_EPSILON.

Reply:
Done! Although I guess it's not so ideal (since we do lose precision during int -> float process :thinking:), but maybe I am just too picky :yum: